src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double legC;
+    private double sideLow;
+    private double sideUp;
+
+    public IsoscelesTrapezoid(String color, double legC, double sideLow, double sideUp) {
+        super(color);
+        this.legC = legC;
+        this.sideLow = sideLow;
+        this.sideUp = sideUp;
+    }
+
+    @Override
+    public double getAreaFigure() {
+        double s = getSemiperimeter();
+        double dd = ((s - sideLow) * (s - sideUp) * Math.pow((s - legC), 2));
+        return Math.sqrt(dd);
+    }
+
+    private double getSemiperimeter() {
+        return 0.5 * (sideUp + sideLow + (2 * legC));
```

Comment:
You can use more simple formula
![image](https://github.com/mate-academy/jv-oop-advanced/assets/63095184/f90b7bbc-e256-4949-888c-1c6781dca6c3)